frontend/src/locales/en.json

```diff
@@ -1,355 +1,358 @@
-    "Today active robots":"Today active robots",
-    "24h non-KYC bitcoin premium":"24h non-KYC bitcoin premium",
-    "Maker fee":"Maker fee",
-    "Taker fee":"Taker fee",
-    "Number of public BUY orders":"Number of public BUY orders",
-    "Number of public SELL orders":"Number of public SELL orders",
-    
+    "BOTTOM BAR AND MISC - BottomBar.js": "Bottom Bar user profile and miscellaneous dialogs",
+    "Stats For Nerds": "Stats For Nerds",
+    "LND version": "LND version",
+    "Currently running commit hash": "Currently running commit hash",
+    "24h contracted volume": "24h contracted volume",
+    "Lifetime contracted volume": "Lifetime contracted volume",
+    "Made with": "Made with",
+    "and": "and",
+    "... somewhere on Earth!": "... somewhere on Earth!",
+    "Community": "Community",
+    "Support is only offered via public channels. Join our Telegram community if you have questions or want to hang out with other cool Robots. Please, use our GitHub Issues if you find a bug or want to see new features!": "Support is only offered via public channels. Join our Telegram community if you have questions or want to hang out with other cool Robots. Please, use our GitHub Issues if you find a bug or want to see new features!",
+    "Join the RoboSats group": "Join the RoboSats group",
+    "Telegram (English / Main)": "Telegram (English / Main)",
+    "RoboSats Telegram Communities": "RoboSats Telegram Communities",
+    "Join RoboSats Spanish speaking community!": "Join RoboSats Spanish speaking community!",
+    "Join RoboSats Russian speaking community!": "Join RoboSats Russian speaking community!",
+    "Join RoboSats Chinese speaking community!": "Join RoboSats Chinese speaking community!",
+    "Join RoboSats English speaking community!": "Join RoboSats English speaking community!",
+    "Tell us about a new feature or a bug": "Tell us about a new feature or a bug",
+    "Github Issues - The Robotic Satoshis Open Source Project": "Github Issues - The Robotic Satoshis Open Source Project",
+    "Your Profile": "Your Profile",
+    "Your Robot": "Your Robot",
+    "Go to active order #{{orderID}}": "Go to active order #{{orderID}}",
+    "Your current order": "Your current order",
+    "No active orders": "No active orders",
+    "Your token (will not remain here)": "Your token (will not remain here)",
+    "Back it up!": "Back it up!",
+    "Cannot remember": "Cannot remember",
+    "Rewards and compensations": "Rewards and compensations",
+    "Share to earn 100 Sats per trade": "Share to earn 100 Sats per trade",
+    "Your referral link": "Your referral link",
+    "Your earned rewards": "Your earned rewards",
+    "Claim": "Claim",
+    "Invoice for {{amountSats}} Sats": "Invoice for {{amountSats}} Sats",
+    "Submit": "Submit",
+    "There it goes, thank you! 🥇": "There it goes, thank you! 🥇",
+    "You have an active order": "You have an active order",
+    "You can claim Sats!": "You can claim Sats!",
```

Comment:
Just remind that any change made to the keys (left side of the ":") will break the translation function. Changes have to be implemented also in "src/components" or discarded.

Reply:
https://github.com/Reckless-Satoshi/robosats/pull/97#discussion_r851772140